src/resources/bootstrap.cpp

```diff
@@ -459,15 +448,31 @@ static bool do_bootstrap(buddy::resources::ProgressHook progress_hook) {
         return false;
     }
 
+    // clear installed resources
+    Path target_path("/internal/res");
+    buddy::resources::InstalledRevision::clear();
+    remove_directory_recursive(target_path);
```

Comment:
What shall we do in case we are unable the delete the directory recursively? Reformat XLFASH? The WD25 chip can have a problem on its own and the user at least needs to know a board replacement is necessary.

Reply:
Yes, reformat and hope in the best. Maybe report it to the user. I plan to add this later 👍